### ISSUE TITLE
plot_tuner() Error: Can't subset columns that don't exist. 

### ISSUE THREAD

**Issue by Pacific-Celt** [henry090/kerastuneR#27]
When passing a nnet in which learning rate is not one of the parameters explored by the tuner, plot_tuner() returns the following error:

```
> result = plot_tuner(tuner)
Error: Can't subset columns that don't exist.
x Column `learning_rate` doesn't exist.
```

**Comment by henry090** [comment]
Thanks for reporting this issue. Could you reinstall and try again, please?

**Comment by Pacific-Celt** [comment]
Reinstalled the dev version and the error is now cleared.
